draft-ietf-rtcweb-jsep.xml

```diff
@@ -3826,23 +3824,38 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                         <t>For each specified payload type that is also
                         supported by the local implementation, establish
-                        a mapping between the payload type ID and the
-                        actual media format. [TODO - Justin to add more
-                        to explain mapping.] If any indicated payload
-                        type is unknown, it MUST be ignored.  [TODO:
-                        should fail on answers]</t>
+                        a mapping between the specified payload type ID and the
+                        actual media format, as described in
+                        <xref target="RFC3264" />, Section 6.1.
+                        Specifically, this means that the
+                        implementation determines the payload type ID to
+                        be used in outgoing RTP packets when sending each
+                        specified payload type. If any indicated payload
+                        type is unknown, it MUST be ignored.</t>
```

Comment:
I'm not sure I understand where you are going with this. It seems like there are two salient properties:

1. The sender of the SDP announces which PTs it wants to use and the associated formats.
2. The answerer also announces which formats it is actually willing to use.

So you need to create:
1. A mapping from the formats you want to use -> PTs you announced for outgoing
2. A mapping from the PTs the other side announced -> the formats they were announced with.

Am I confused?